probe-dictionary/explore.js

```diff
@@ -461,7 +471,7 @@ function renderMeasurements(measurements) {
     ["name", (d, h, c) => d.name],
     ["type", (d, h, c) => d.type],
     ["population", (d, h, c) => h.optout ? "release" : "prerelease"],
-    ["recorded", (d, h, c, history) => friendlyRecordingRangeForHistory(history, c)],
+    ["recorded", (d, h, c, history) => friendlyRecordingRangeForHistory(history, c, false)],
```

Comment:
nit: should we add `/* filterPrelrease */` after `false`? It would make it easier to understand without having tor scroll the file back to the function definition

Reply:
That makes these lines pretty long though, i'm torn on that.

Reply:
Feel free to drop this and the one below!